### PR TITLE
Fix Part of #4883 : Remove exempted file from Kotlin and Xml Regex components colors checks

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fix Part of #4883 : Remove exempted file from Kotlin and Xml Regex components colors checks

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by MohitGupta121 — COMMENTED** [review]
@BenHenning PTAL, I make changes and remove exempted files from Regex check that we dicussed here https://github.com/oppia/oppia-android/pull/4882#issuecomment-1468775083

File: app/src/main/res/values-night/color_palette.xml [hunk 1]
```diff
@@ -200,4 +202,29 @@
   <color name="color_palette_onboarding_4_status_bar_color">@color/color_def_onboarding_4_status_bar</color>
   <color name="color_palette_coming_soon_rect_background_end_color">@color/color_def_grape_violet</color>
   <color name="color_palette_coming_soon_rect_background_start_color">@color/color_def_bright_violet</color>
+  <!-- AVATAR BACKGROUND COLORS -->
```
Comment (BenHenning) [inline app/src/main/res/values-night/color_palette.xml]: Do these need to be defined here if they're already defined in -day? I thought we only needed -night versions if they are different.

Ditto for all other colors in this XML file (including ones already checked in).

File: app/src/main/res/values/color_palette.xml [hunk 2]
```diff
@@ -203,4 +205,29 @@
   <color name="color_palette_onboarding_4_status_bar_color">@color/color_def_onboarding_4_status_bar</color>
   <color name="color_palette_coming_soon_rect_background_end_color">@color/color_def_grape_violet</color>
   <color name="color_palette_coming_soon_rect_background_start_color">@color/color_def_bright_violet</color>
+  <!-- AVATAR BACKGROUND COLORS -->
+  <color name="color_palette__avatar_background_1_color">@color/color_def_avatar_background_1</color>
```
Comment (BenHenning) [inline app/src/main/res/values/color_palette.xml]: Nit: there's an extra underscore in the middle of all of these new color names--please fix.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @MohitGupta121! I didn't find much, just a few small things. PTAL.

File: app/src/main/res/values/color_palette.xml (hunk 2 above)
Comment (MohitGupta121) [reply app/src/main/res/values/color_palette.xml]: Done.

File: app/src/main/res/values-night/color_palette.xml (hunk 1 above)
Comment (MohitGupta121) [reply app/src/main/res/values-night/color_palette.xml]: Done.

**Review by MohitGupta121 — COMMENTED** [review]
Thanks @BenHenning, PTAL, updated the review changes.

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121, this LGTM! Approving & enabling auto-merge.

**Pull request merged**